<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget{
background-color:#E3EAF4;
}
QLineEdit{
background-color:#E3EAF4;
}
QDialog{
background-color:#E3EAF4;
margin-top:10px;
}
QLabel{
font-size:14px;
font-family:Microsoft YaHei;
background-color:#E3EAF4;
}
QGroupBox{
font-size:14px;
font-family:Microsoft YaHei;
border:1px solid rgb(0, 0, 0);
background-color:#E3EAF4;
}
QPushButton{
background-color:#2884D8;
color: #FFFFFF;
font-size:14px;
font-family:Microsoft YaHei;
}
QTextBrowser{
border-width:0;
border-style:outset;
background-color:#E3EAF4;
}
QGraphicsView{
background-color:#E3EAF4;
border-width:0;
border-style:outset;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <family>Microsoft YaHei</family>
                <pointsize>-1</pointsize>
               </font>
              </property>
              <property name="text">
               <string>IP地址</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_ip_address"/>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="pushButton_ping">
              <property name="text">
               <string>ping</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>提示信息</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <property name="topMargin">
              <number>20</number>
             </property>
             <item row="0" column="0">
              <widget class="QTextBrowser" name="textBrowser_tips"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton_next">
              <property name="text">
               <string>下一步</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>日志</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <property name="topMargin">
            <number>20</number>
           </property>
           <item row="0" column="0">
            <widget class="QTextBrowser" name="textBrowser_log">
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../iconQrc.qrc"/>
 </resources>
 <connections/>
</ui>
